.storybook/config.js

```diff
@@ -4,24 +4,22 @@ import {configure, addDecorator} from '@storybook/react';
 import {Provider as StyletronProvider} from 'styletron-react';
 import {Client as Styletron} from 'styletron-engine-atomic';
 
-import {LayersManager} from '../src/layer';
-import {ThemeProvider} from '../src/styles';
+import {BaseProvider} from '../src/index';
 import {LightThemeMove, DarkThemeMove} from '../src/themes';
 
 // Add providers for theme and styletron
 const engine = new Styletron();
 addDecorator((story, context) => {
   return (
     <StyletronProvider value={engine}>
-      <LayersManager>
-        <ThemeProvider
-          theme={
-            context.kind === 'baseui-dark' ? DarkThemeMove : LightThemeMove
-          }
-        >
-          {story()}
-        </ThemeProvider>
-      </LayersManager>
+      <BaseProvider
+        themeProvider={{
```

Comment:
why not just 

```jsx
<BaseProvider theme={...} />
```

there are no other props that ThemeProvider accepts anyway?

Reply:
In case we decide to extend the ThemeProvider 🤷‍♀ Or if there are more providers to be added that have more than one prop, so it's consistent.

Reply:
I think the api to this should be designed so that consumers do not need to know that there are multiple 'providers' bundled into one. if there needs to be a new value passed through, we can add it to the top level props

Reply:
I agree, just using the top level `theme` prop seems like a cleaner API